system-test/spanner.ts

```diff
@@ -9191,6 +9191,92 @@ describe('Spanner', () => {
         commitTransaction(done, PG_DATABASE, postgreSqlTable);
       });
 
+      describe('when multiplexed session is enabled for read write', async () => {
```

Comment:
This need not be specific to multiplexed session, the scenario we are trying to test here is that parallel transaction should work , for both mux and non-mux. So describe should mention that.